modules/psap-special-resource-operator-using-CRD.adoc

```diff
@@ -0,0 +1,451 @@
+            apiVersion: policy.open-cluster-management.io/v1
+            kind: ConfigurationPolicy
+            metadata:
+                name: config-{{.Values.specialResourceModule.metadata.name}}-ds
+            spec:
+                remediationAction: enforce
+                severity: low
+                namespaceselector:
+                    exclude:
+                        - kube-*
+                    include:
+                        - '*'
+                object-templates:
+                    - complianceType: musthave
+                        objectDefinition:
+                        apiVersion: v1
+                        kind: Namespace
+                        metadata:
+                            name: {{.Values.specialResourceModule.spec.namespace}}
+                    - complianceType: mustonlyhave
+                        objectDefinition:
+                        apiVersion: v1
+                        kind: ServiceAccount
+                        metadata:
+                            name: {{.Values.specialResourceModule.metadata.name}}
+                            namespace: {{.Values.specialResourceModule.spec.namespace}}
+                    - complianceType: mustonlyhave
+                        objectDefinition:
+                            apiVersion: rbac.authorization.k8s.io/v1
+                            kind: Role
+                            metadata:
+                                name: {{.Values.specialResourceModule.metadata.name}}
+                                namespace: {{.Values.specialResourceModule.spec.namespace}}
+                            rules:
+                                - apiGroups:
+                                    - security.openshift.io
+                                resources:
+                                    - securitycontextconstraints
+                                verbs:
+                                    - use
+                                resourceNames:
+                                    - privileged
+                    - complianceType: mustonlyhave
+                        objectDefinition:
+                            apiVersion: rbac.authorization.k8s.io/v1
+                            kind: RoleBinding
+                            metadata:
+                                name: {{.Values.specialResourceModule.metadata.name}}
+                                namespace: {{.Values.specialResourceModule.spec.namespace}}
+                            roleRef:
+                                apiGroup: rbac.authorization.k8s.io
+                                kind: Role
+                                name: {{.Values.specialResourceModule.metadata.name}}
+                            subjects:
+                                - kind: ServiceAccount
+                                name: {{.Values.specialResourceModule.metadata.name}}
+                                namespace: {{.Values.specialResourceModule.spec.namespace}}
```

Comment:
These should be at the same level as `kind`

Reply:
corrected